core-war/src/main/webapp/attachment/jsp/displayAttachedFiles.jsp

```diff
@@ -1335,6 +1313,24 @@ const _afManager${domIdSuffix} = new function() {
 };
 </script>
 
+<c:if test="${empty __addAttachmentDialogInserted}">
+  <c:set var="__addAttachmentDialogInserted" value="${true}" scope="request"/>
+  <silverpeas-attachment-management id="attachmentManagement"
+                                    v-on:api="manager = $event"></silverpeas-attachment-management>
+  <script type="text/javascript">
+    whenSilverpeasReady(function() {
+      window.attachmentVm = new Vue({
```

Comment:
Any VueJS developer will immediately understand that vm is about VuesJS because it is a convention introduced by VueJS itself (2.x).
VM means Vue Model.
When migrating to 3.x, maybe it will be a renaming because of change about how to start VueJS management in a page.

Reply:
Ok